actors/builtin/multisig/multisig_state.go

```diff
@@ -55,57 +53,15 @@ func (st *State) assertAvailable(currBalance abi.TokenAmount, amountToSpend abi.
 	return nil
 }
 
-func (as *State) getPendingTransaction(s adt.Store, txnID TxnID) (Transaction, error) {
-	hm, err := adt.AsMap(s, as.PendingTxns)
-	if err != nil {
-		return Transaction{}, err
-	}
-
+func getPendingTransaction(ptx *adt.Map, txnID TxnID, cid cid.Cid) (Transaction, error) {
 	var out Transaction
-	found, err := hm.Get(txnID, &out)
+	found, err := ptx.Get(txnID, &out)
 	if err != nil {
 		return Transaction{}, errors.Wrapf(err, "failed to read transaction")
 	}
 	if !found {
-		return Transaction{}, errors.Errorf("failed to find transaction %v in HAMT %s", txnID, as.PendingTxns)
+		return Transaction{}, errors.Errorf("failed to find transaction %v in HAMT %s", txnID, cid)
```

Comment:
Don't take the CID parameter just for this message. Actually using it for something meaningful would probably be a bug.